src/constructRoutes.js

```diff
@@ -78,38 +94,21 @@ function validate(routesConfig) {
         disableWarnings
       );
       validateString(`${propertyName}.path`, route.path);
-      if (route.hasOwnProperty("routes")) {
-        validateArray(`${propertyName}.routes`, route.routes, validateRoute);
-      }
+      validateArray(`${propertyName}.routes`, route.routes, validateRoute);
     }
   }
-}
 
-function getBaseURI() {
```

Comment:
Turns out that we don't need to worry about the base URL of the page, only the base URL of our router. At least I think so, based on how React Router works